solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -702,6 +700,28 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
         return _conversionFee.mul(y).div(x.mul(AMPLIFICATION_FACTOR).add(y).sub(y.mul(AMPLIFICATION_FACTOR)));
     }
 
+    /**
+      * @dev creates the converter's pool tokens
+      * note that technically pool tokens can be created on deployment but gas limit
+      * might get too high for a block, so creating them on first activation
+      *
+    */
+    function createPoolTokens() internal {
+        IPoolTokensContainer container = IPoolTokensContainer(anchor);
+        if (container.poolTokens().length != 0) {
+            return;
+        }
+
+        uint256 reserveCount = reserveTokens.length;
+        for (uint256 i = 0; i < reserveCount; i++) {

```

Comment:
I would change here to `uint256 i = container.poolTokens().length...` and get rid of the check above (same effect but cleaner code).